dynamic_programming/bitmask.py

```diff
@@ -9,77 +9,85 @@
 Find the total no of ways in which the tasks can be distributed.
 """
 from collections import defaultdict
+from typing import List, Dict
```

Comment:
These are not needed in Python 3.12 (or with `from __future__ import annotations`), just use `list` and `dict`.